src/Symfony/Component/PropertyInfo/Tests/Extractor/ConstructorExtractorTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Symfony\Component\PropertyInfo\Tests\Extractor;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\PropertyInfo\Extractor\ConstructorExtractor;
+use Symfony\Component\PropertyInfo\Tests\Fixtures\DummyExtractor;
+use Symfony\Component\PropertyInfo\Type;
+
+/**
+ * @author Dmitrii Poddubnyi <dpoddubny@gmail.com>
+ */
+class ConstructorExtractorTest extends TestCase
+{
+    /**
+     * @var ConstructorExtractor
+     */
+    private $extractor;
+
+    protected function setUp()
+    {
+        $this->extractor = new ConstructorExtractor([new DummyExtractor()]);
+    }
+
+    public function testInstanceOf()
+    {
+        $this->assertInstanceOf('Symfony\Component\PropertyInfo\PropertyTypeExtractorInterface', $this->extractor);
+    }
+
+    public function testGetTypes()
+    {
+        $this->assertEquals([new Type(Type::BUILTIN_TYPE_STRING)], $this->extractor->getTypes('Foo', 'bar', []));
```

Comment:
Can you use `assertSame()` here?

Reply:
I can't, assertSame `asserts that two variables reference the same object.`
![image](https://user-images.githubusercontent.com/1675033/53183746-950df600-3604-11e9-9b31-5234ced7273b.png)